test/util/testnode/full_node.go

```diff
@@ -147,8 +147,8 @@ func DefaultTendermintConfig() *config.Config {
 	// during tests.
 	tmCfg.Consensus.TargetHeightDuration = 300 * time.Millisecond
 	tmCfg.Consensus.TimeoutPropose = 200 * time.Millisecond
-	tmCfg.Mempool.MaxTxBytes = 220020096 // 210MB
-	tmCfg.RPC.MaxBodyBytes = 220020096   // 210MB
+	tmCfg.Mempool.MaxTxBytes = 220_000_960 // 210MB
+	tmCfg.RPC.MaxBodyBytes = 220_000_960   // 210MB
```

Comment:
This is 200_000 bytes less than 210 MB

```suggestion
	tmCfg.Mempool.MaxTxBytes = 220_200_960 // 210MB
	tmCfg.RPC.MaxBodyBytes = 220_200_960   // 210MB
```

Ref: https://www.flightpedia.org/convert/210-megabytes-to-bytes.html

Reply:
damn I actually tried to copy from the site you linked there to be sure but messed up apparently :sweat_smile: 

as mentioned in a different comment, this value is _purlely_ to remove any barriers for being able to submit transactions to test nodes, there is no rhyme or reason to it besides it being absolutely obtuse. 

therefore I'm just using powers of ten like god intended and the max square size
https://github.com/celestiaorg/celestia-app/pull/1772/commits/7334b9005c81576a848c4e75c66c289eba0e5957

Reply:
I thought MB was metric! the above suggestion confused me, also is that site incorrect since it gives the same value for mega and mebi?

https://github.com/celestiaorg/celestia-app/pull/1772#discussion_r1198176298

Reply:
oof I think that site is broken. According to [this post](https://www.majordifferences.com/2018/03/differences-between-megabyte-and.html):

> [Home](https://www.majordifferences.com/)[computer](https://www.majordifferences.com/search/label/computer)
Differences between Megabyte and Mebibyte (MB vs MiB)
[3 Comments](https://www.majordifferences.com/2018/03/differences-between-megabyte-and.html#comments)
[Facebook](javascript:;)
[Twitter](javascript:;)
Both Megabyte (MB) and Mebibyte (MiB) are units of information or computer storage. But MB is used in two ways either as equivalent to 1000 Kb or 1024 Kb. This creates some sort of confusion. To avoid this, the International Electrotechnical Commission (IEC) (1998), the leading international organization for worldwide standardization in electrotechnology, approved as an IEC International Standard names and symbols for prefixes for binary multiples for use in the fields of data processing and data transmission. They recommended the use of Mebibyte (a combination of Mega and Binary) MiB, which always refer to 1024 Kb.

Proposal to always use the MiB variant to avoid confusion